### PR TITLE
Fix slovnaft-bajk

### PR DESCRIPTION
**Pull request by martgnz**
Closes https://github.com/eskerda/pybikes/issues/598.

### CODE REVIEW

File: pybikes/slovnaft_bajk.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+from pybikes.base import BikeShareSystem, BikeShareStation
+from pybikes import utils
+
+FEED_URL = "https://gate.slovnaftbajk.sk/AppGate2.php"
+
+
+class SlovnaftBajk(BikeShareSystem):
+    def __init__(self, tag, meta):
+        super(SlovnaftBajk, self).__init__(tag, meta)
+
```
Comment (eskerda) [inline pybikes/slovnaft_bajk.py:16]:
```suggestion
```

File: pybikes/slovnaft_bajk.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2023, Martín González Gómez <m@martingonzalez.net>
+# Distributed under the AGPL license, see LICENSE.txt
+
+import json
+
+from pybikes.base import BikeShareSystem, BikeShareStation
```
Comment (eskerda) [inline pybikes/slovnaft_bajk.py:7]:
```suggestion
from pybikes.base import BikeShareSystem, BikeShareStation, PyBikesScraper
```

File: pybikes/slovnaft_bajk.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+from pybikes import utils
```
Comment (eskerda) [inline pybikes/slovnaft_bajk.py:8]:
```suggestion
```

File: pybikes/slovnaft_bajk.py [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+FEED_URL = "https://gate.slovnaftbajk.sk/AppGate2.php"
+
+
+class SlovnaftBajk(BikeShareSystem):
+    def __init__(self, tag, meta):
+        super(SlovnaftBajk, self).__init__(tag, meta)
+
+    def update(self, scraper=None):
+        scraper = scraper or utils.PyBikesScraper()
```
Comment (eskerda) [inline pybikes/slovnaft_bajk.py:18]:
```suggestion
        scraper = scraper or PyBikesScraper()
```

**Review by eskerda — CHANGES_REQUESTED** [review]

**Comment by martgnz** [comment]
Thanks for your review. I copied from an old system that had all those imports. Will remember for the next one.

**Pull request merged**